tests/unit/vcast-server.test.ts

```diff
@@ -95,7 +98,7 @@ describe("test server functions", () => {
     );
   });
 
-  it("serverIsAlive handles Python interface error", async () => {
+  test("serverIsAlive handles Python interface error", async () => {
```

Comment:
Can we revert to `it` for consistency with other tests? If we want to switch from `test` to `it`, it should be a separate PR that changes this in all tests inside `unit`